<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="maximumSize">
        <size>
         <width>799</width>
         <height>489</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_scan">
        <attribute name="title">
         <string>Scan</string>
        </attribute>
        <widget class="QTreeView" name="treeView">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>251</width>
           <height>411</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>40</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Rod</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Scanned folders:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_number_dir">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>40</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>90</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Rod</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>   Scanned files:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>500</x>
           <y>140</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>140</y>
           <width>181</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Rod</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Found:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_number_file">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>90</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_number_viruse">
         <property name="geometry">
          <rect>
           <x>480</x>
           <y>140</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
        <widget class="QProgressBar" name="progressBar">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>270</y>
           <width>491</width>
           <height>23</height>
          </rect>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>650</x>
           <y>400</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Scan!</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>650</x>
           <y>430</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_selected_drive">
         <property name="geometry">
          <rect>
           <x>290</x>
           <y>190</y>
           <width>171</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Rod</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_scanResults">
        <attribute name="title">
         <string>ScanResults</string>
        </attribute>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>430</x>
           <y>430</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>540</x>
           <y>430</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Delete </string>
         </property>
        </widget>
        <widget class="QScrollArea" name="scrollArea_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>801</width>
           <height>411</height>
          </rect>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>799</width>
            <height>409</height>
           </rect>
          </property>
          <widget class="QListView" name="listView">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>791</width>
             <height>411</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>791</width>
             <height>411</height>
            </size>
           </property>
          </widget>
         </widget>
        </widget>
        <widget class="QPushButton" name="pushButton_5">
         <property name="geometry">
          <rect>
           <x>650</x>
           <y>430</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_database">
        <attribute name="title">
         <string>Database</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>791</width>
           <height>461</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>787</width>
               <height>457</height>
              </rect>
             </property>
             <widget class="QPushButton" name="pushButton_6">
              <property name="geometry">
               <rect>
                <x>640</x>
                <y>420</y>
                <width>75</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>Exit</string>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_7">
              <property name="geometry">
               <rect>
                <x>640</x>
                <y>40</y>
                <width>75</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
             <widget class="QScrollArea" name="scrollArea_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>481</width>
                <height>461</height>
               </rect>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_2">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>479</width>
                 <height>459</height>
                </rect>
               </property>
               <widget class="QTextEdit" name="textEdit">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>481</width>
                  <height>461</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>Khmer UI</family>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                  <underline>true</underline>
                 </font>
                </property>
               </widget>
              </widget>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>679</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>600</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>679</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>683</x>
     <y>534</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>677</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>525</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
